org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java

```diff
@@ -378,4 +381,26 @@ public static XMLFormattingOptions create(FormattingOptions options, FormattingO
 		return new XMLFormattingOptions(options).merge(sharedFormattingOptions);
 	}
 
+	/**
+	 * A setting for enabling the XML formatter to move the closing bracket of a tag with at least 2 attributes
+	 * to a new line.
+	 *
+	 * Returns the value of closingBracketNewLine or false if it was set to null
```

Comment:
Swap the order of these comments. Your IDE/editor will display the first sentence as the main documentation, and the rest as the detailed documentation.